### ISSUE TITLE
[Feature Request] Portable version is Not Portable?

### ISSUE THREAD

**Issue by o-data** [OneFolderApp/OneFolder#95]
**Describe the bug**
With the Portable version of the release, is not really Portable in a sense the setting is saved within the directory/folder. It seem to still use %appdata% (for Windows) or Registry to store setting. Meaning we can't really use this on a USB stick.

Example of Portable Software: https://portableapps.com/

All setting and data and the software is stored in the Folder so we can use it on a USB.

**To Reproduce**
1. Download Portable version of OneFolder
2. Create two Folder, copy the download exe to each folder
3. Run OneFolder, use it however you want a while. Close it.
4. Run Onefolder from 2nd Folder. All setting is already saved and you are using the same OneFolder data and setting.

**Expected behavior**
Portable to be Portable. Data is not share.

**Screenshots**

**OneFolder version**

- e.g. v1.0.11

**Desktop OS**
Windows 11

**Additional context**
Thank you

**Comment by Antoine-lb** [comment]
This is true, is not really portable. It uses outside folder to store the settings, db backups, thumnbnails and other stuff.

We like the idea of a truly portable app, but don't expect it to arrive soon, there are some ground work to do.

Behind the curtains we are planing on moving away from using IndexDB for SQLite. This will probably mark the V2 milestone.

This SQLite file will probably be store in the same folder as the files it's reading. This could make OneFolder much closer to be truly portable, but this are only plans for now, nothing concrete.
